client/src/dashboard/Dashboard.js

```diff
@@ -246,6 +247,7 @@ function DashboardContent() {
                       <Route path="/formulator" element={<RuleFormulator />} />
                       <Route path="/jdbc" element={<JDBCDrivers />} />
                       <Route path="/queries" element={<QueryLogs />} />
+                      <Route exact path="/manageapps" element={<ManageApplication />} />
```

Comment:
Same here, rename `/manageapps` to `/apps` or `/applications`, rename `ManageApplication` to `Applications` if no conflicts.